### DISCUSSION TITLE
Categories/Tags Title Not Displaying After Deploy

### DISCUSSION THREAD

**Discussion by gshick** [CaiJimmy/hugo-theme-stack#850] [General]
Working on this site: https://[support-our-stem.netlify.app](https://support-our-stem.netlify.app/)/

When I render locally, the words "categories" and "tags" appear above the tag cloud on the right bar, next to the respective images.

However, when I push to Github and deploy via netlify, they disappear.  Any ideas on how to include the titles?  Upon inspection, the related HTML is widget-title section-title.  Any assistance is appreciated.

**Comment by Lovirr** [comment]
that's so weried...
never heard this before and it does not happen in the example site given by the author , maybe you should check out your hugo version?
